install-dev/upgrade/php/ps_1760_replace_index_of_admin_filter.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+function ps_1760_replace_index_of_admin_filter()
```

Comment:
maybe we should create functions because this kind of scenario might happen again in the future, so having a `remove_table_index($table, $indexName)` and `add_table_index($table, $indexName)` might be useful

Reply:
You're right, like we did for tab management.

Reply:
yep that's the idea

Reply:
I actually did not create a method for index creation, as their signature will always be different. Only the deletion has been included in the last commit.